core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-types-fields-list/content-types-fields-list.component.html

```diff
@@ -1,13 +1,12 @@
 <ul
-    [dragulaModel]="fieldTypes"
+    [dragulaModel]="$fieldTypes()"
     class="content-types-fields-list"
     dragula="fields-bag"
     data-drag-type="source">
-    <li
-        *ngFor="let fieldType of fieldTypes"
-        [attr.data-clazz]="fieldType.clazz"
-        class="content-types-fields-list__item">
-        <dot-icon name="{{ fieldService.getIcon(fieldType.clazz) }}"></dot-icon>
-        <span role="listitem">{{ fieldType.name }}</span>
-    </li>
+    @for (fieldType of $fieldTypes(); track fieldType.clazz) {
+        <li [attr.data-clazz]="fieldType.clazz" class="content-types-fields-list__item">
+            <dot-icon name="{{ getIcon(fieldType.clazz) }}"></dot-icon>
```

Comment:
I think it is better to prepare the data; instead, call a method directly in the template.

Reply:
agree, changed.